www/novj35/ukol_4/css/styles-page-layout.css

```diff
@@ -0,0 +1,104 @@
+* {
+    box-sizing: border-box;
+  }
+  body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+}
+
+.clear {
+  clear: both;
+}
+
+h1 {
+  color: white;
+  text-align: center;
+  padding: 50px 10px;
+  background-color: rgb(192, 50, 50); 
+}
+
+.instructions {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 20px;
+  margin-bottom: 50px;
+}
+
+.colors>input {
+  color: white;
+  width: 50%;
+  padding: 5px;
+  border: none;
+  font-size: 18px;
+  margin: 0;
+  float: left;
+  
+}
+
+.wrapper {
+  color: white;
+  font-size: 35px;
+  text-align: center;
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 20px;
+  margin-bottom: 50px;
+}
+
+header{
+  width: 100%;
+  background-color: rgb(223, 114,  75);
+  height: 200px;
+  padding: 75px;
+}
+
+nav{
+  width: 100%;
+  background-color: rgb( 47, 108, 136);
+  height: 80px;
+  padding: 20px;
+}
+
+main {
+  width: 75%;
+  background-color: rgb(192, 70, 94);
+  height: 700px; 
+  padding: 300px;
```

Comment:
padding i na obe strany zpusobuje to, ze se element nevejde do mensi obrazovky

co takhle rychly test responzivity (resizovat okno) pred odevzdanim?